README.md

```diff
@@ -153,13 +221,49 @@ import { clearError } from '@americanexpress/one-app-ducks';
 store.dispatch(clearError());
 ```
 
-### `errorReporting`
+### `errorReporting` Duck
+
+The error reporting duck is for sending client side errors to the `errorReportingUrl` configured by
+the  environment variable `ONE_CLIENT_REPORTING_URL`. You can find more documentation on environment
+variables for One App in the [One App documentation](https://github.com/americanexpress/one-app/blob/master/runtime-configuration.md).
+On the server the errors reported are simply logged. Reported errors will have the following format:
```

Comment:
`...simply logged with the assumption that the underlying infrastructure will pick up those logs and ship them to where they can be better kept and analyzed`.